app/code/Magento/Catalog/Controller/Adminhtml/Category/Edit.php

```diff
@@ -115,10 +120,6 @@ public function execute()
      */
     private function getStoreManager()
```

Comment:
It would be better to remove this method and fix all usages to newly injected `$this->storeManager`

Reply:
Fixed, thanks.